content/300-accelerate/200-getting-started.mdx

```diff
@@ -20,7 +20,9 @@ To get started with Accelerate, you will need the following:
 
 ## 1. Enable Accelerate in a project
 
-In order to enable Accelerate, you can log in to [Prisma Data Platform](https://pris.ly/pdp) and create a new project. Follow the instructions in the UI to add Accelerate.
+In order to enable Accelerate, you can log in to [Prisma Data Platform](https://pris.ly/pdp) and create a new project. If your database uses IP allowlisting or you're setting up a firewall to restrict access to trusted IP addresses, ensure static IPs are enabled in your Accelerate configuration.
+
+> ℹ️ To enable static IP support for Accelerate within your existing or new project environment, ensure you're on our **Pro** or **Business** plan. Refer to our [pricing page](https://www.prisma.io/pricing) for more information. To learn more about static IP see this [FAQ](/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate).
```

Comment:
```suggestion
> ℹ️ To enable static IP support for Accelerate within your existing or new project environment, you will need to be on our **Pro** or **Business** plans. Take a look at the [pricing page](https://www.prisma.io/pricing) for more information.
```

I'd move "To learn more about static IP see this [FAQ](/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate)." to the previous paragraph. And I don't think we need a callout for the pricing part, it puts too much emphasis on it.